### ISSUE TITLE
SmartCity-Accessibility Server Side Network Tests #40 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#363]
Build 'SmartCity-Accessibility Server Side Network Tests' is failing!

Last 50 lines of build :
```
[...truncated 5.24 KB...]
 :: Spring Boot ::        (v1.5.2.RELEASE)

-- Product name: JxMaps
-- Licensed version: 1.x
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: May 22, 2017
JxMaps license valid.
Running smartcity.accessibility.services.NavigationServiceTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 24.019 sec - in smartcity.accessibility.services.NavigationServiceTest
Running smartcity.accessibility.navigation.NavigationTest
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.3 sec - in smartcity.accessibility.navigation.NavigationTest
Running smartcity.accessibility.database.ParseDatabaseTest
Tests run: 6, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 2.957 sec <<< FAILURE! - in smartcity.accessibility.database.ParseDatabaseTest
testGetLocation(smartcity.accessibility.database.ParseDatabaseTest)  Time elapsed: 0.644 sec  <<< FAILURE!
java.lang.AssertionError: couldn't find the sample object
	at smartcity.accessibility.database.ParseDatabaseTest.testGetLocation(ParseDatabaseTest.java:85)


Results :

Failed tests: 
  ParseDatabaseTest.testGetLocation:85 couldn't find the sample object

Tests run: 13, Failures: 1, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:36 min
[INFO] Finished at: 2017-05-22T08:54:12+00:00
[INFO] Final Memory: 32M/155M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [KoralChapnik] 580fa9f144bc1fc6ab38f9e008455c372f32519b - #333

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Network%20Tests/40/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Network Tests' is failing!

Last 50 lines of build :
```
[...truncated 5.23 KB...]
 :: Spring Boot ::        (v1.5.2.RELEASE)

-- Product name: JxMaps
-- Licensed version: 1.x
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: May 22, 2017
JxMaps license valid.
Running smartcity.accessibility.services.NavigationServiceTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 21.119 sec - in smartcity.accessibility.services.NavigationServiceTest
Running smartcity.accessibility.navigation.NavigationTest
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.751 sec - in smartcity.accessibility.navigation.NavigationTest
Running smartcity.accessibility.database.ParseDatabaseTest
Tests run: 6, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 3.242 sec <<< FAILURE! - in smartcity.accessibility.database.ParseDatabaseTest
testGetLocation(smartcity.accessibility.database.ParseDatabaseTest)  Time elapsed: 0.664 sec  <<< FAILURE!
java.lang.AssertionError: couldn't find the sample object
	at smartcity.accessibility.database.ParseDatabaseTest.testGetLocation(ParseDatabaseTest.java:85)


Results :

Failed tests: 
  ParseDatabaseTest.testGetLocation:85 couldn't find the sample object

Tests run: 13, Failures: 1, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:30 min
[INFO] Finished at: 2017-05-22T09:00:00+00:00
[INFO] Final Memory: 33M/207M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [KoralChapnik] 580fa9f144bc1fc6ab38f9e008455c372f32519b - #333

- [Arthur Sapozhnikov] aa0c70d17d5e0bbed998a3f37bb7e163a1eebcda - Fixed token #358



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Network%20Tests/41/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Network Tests' is failing!

Last 50 lines of build :
```
[...truncated 5.23 KB...]
 :: Spring Boot ::        (v1.5.2.RELEASE)

-- Product name: JxMaps
-- Licensed version: 1.x
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: May 22, 2017
JxMaps license valid.
Running smartcity.accessibility.services.NavigationServiceTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 20.522 sec - in smartcity.accessibility.services.NavigationServiceTest
Running smartcity.accessibility.navigation.NavigationTest
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.498 sec - in smartcity.accessibility.navigation.NavigationTest
Running smartcity.accessibility.database.ParseDatabaseTest
Tests run: 6, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 2.887 sec <<< FAILURE! - in smartcity.accessibility.database.ParseDatabaseTest
testGetLocation(smartcity.accessibility.database.ParseDatabaseTest)  Time elapsed: 0.545 sec  <<< FAILURE!
java.lang.AssertionError: couldn't find the sample object
	at smartcity.accessibility.database.ParseDatabaseTest.testGetLocation(ParseDatabaseTest.java:85)


Results :

Failed tests: 
  ParseDatabaseTest.testGetLocation:85 couldn't find the sample object

Tests run: 13, Failures: 1, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:29 min
[INFO] Finished at: 2017-05-22T09:09:50+00:00
[INFO] Final Memory: 33M/208M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [KoralChapnik] 580fa9f144bc1fc6ab38f9e008455c372f32519b - #333

- [Arthur Sapozhnikov] aa0c70d17d5e0bbed998a3f37bb7e163a1eebcda - Fixed token #358

- [Ariel Kolikant] 6c30dc28bff403b43c89bb6c84dc3c27109199fd - token in userinformationservice#358



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Network%20Tests/42/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Network Tests' is failing!

Last 50 lines of build :
```
[...truncated 6.05 KB...]
 :: Spring Boot ::        (v1.5.2.RELEASE)

-- Product name: JxMaps
-- Licensed version: 1.x
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: May 22, 2017
JxMaps license valid.
Running smartcity.accessibility.services.NavigationServiceTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 23.861 sec - in smartcity.accessibility.services.NavigationServiceTest
Running smartcity.accessibility.navigation.NavigationTest
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.526 sec - in smartcity.accessibility.navigation.NavigationTest
Running smartcity.accessibility.database.ParseDatabaseTest
Tests run: 6, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 3.167 sec <<< FAILURE! - in smartcity.accessibility.database.ParseDatabaseTest
testGetLocation(smartcity.accessibility.database.ParseDatabaseTest)  Time elapsed: 0.63 sec  <<< FAILURE!
java.lang.AssertionError: couldn't find the sample object
	at smartcity.accessibility.database.ParseDatabaseTest.testGetLocation(ParseDatabaseTest.java:85)


Results :

Failed tests: 
  ParseDatabaseTest.testGetLocation:85 couldn't find the sample object

Tests run: 13, Failures: 1, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:37 min
[INFO] Finished at: 2017-05-22T09:23:08+00:00
[INFO] Final Memory: 33M/204M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [KoralChapnik] 580fa9f144bc1fc6ab38f9e008455c372f32519b - #333

- [Arthur Sapozhnikov] aa0c70d17d5e0bbed998a3f37bb7e163a1eebcda - Fixed token #358

- [Ariel Kolikant] 6c30dc28bff403b43c89bb6c84dc3c27109199fd - token in userinformationservice#358

- [Arthur Sapozhnikov] bac56bbf1aa0eff289a6a9ff58a224563d1e30e6 - Forgot to add locataion

- [Arthur Sapozhnikov] 20bc0d6f0ea5bb4e6f2ff7478bf3ff20a7a0ca66 - Basic implementation of the server side of #256

- [Ariel Kolikant] aa79ef8aa3454310f57afa56ffc9452e25717bfe - #309 added the service for locations in radius

- [Ariel Kolikant] c16a0b5d4c8744c3cfbd8b01ca545bb093e0e287 - #309 git file wasn't added



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Network%20Tests/43/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
